images/ceph/Makefile

```
@@ -22,7 +22,8 @@ CEPH_VERSION = v16.2.5-20210708
 else
 CEPH_VERSION = v16.2.5-20210708
 endif
-BASEIMAGE = ceph/ceph-$(GOARCH):$(CEPH_VERSION)
+REGISTRY_NAME = quay.io
+BASEIMAGE = $(REGISTRY_NAME)/ceph/ceph-$(GOARCH):$(CEPH_VERSION)
```

nit: since the registry name var isn't used elsewhere, what if we just keep it inline for easier searching?
```suggestion
BASEIMAGE = quay.io/ceph/ceph-$(GOARCH):$(CEPH_VERSION)
```

---

One could decide to run make with `REGISTRY_NAME=docker.io make build` so I thought it would be nice to have and doesn't hurt.